packages/@ourworldindata/core-table/src/Transforms.ts

```diff
@@ -17,10 +17,12 @@ import {
 import * as R from "remeda"
 
 enum TransformParamType {
-    TimeSlug = "TimeSlug", // column with time
-    EntitySlug = "EntitySlug", // column with entity
-    DataSlug = "DataSlug", // column with data
-    ColumnSlug = "ColumnSlug", // any column
```

Comment:
I got rid of this one because it was a bit annoying that it can be anything (time, entity or data)